plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/view/support/SupportZip.js

```diff
@@ -35,6 +35,7 @@ Ext.define('NX.coreui.view.support.SupportZip', {
       xtype: 'nx-settingsform',
       settingsFormSubmitMessage: NX.I18n.get('ADMIN_SUPPORT_ZIP_CREATING'),
       settingsFormSuccessMessage: NX.I18n.get('ADMIN_SUPPORT_ZIP_CREATED'),
+      timeout: 60 * 3, // 3 minutes
```

Comment:
this is in seconds to be clear (timeout of provider is in millis) but form bits convert seconds to millis.

For ref:

http://docs-origin.sencha.com/extjs/4.2.3/#!/api/Ext.form.Panel-cfg-timeout
http://docs-origin.sencha.com/extjs/4.2.3/#!/api/Ext.form.Basic-cfg-timeout
http://docs-origin.sencha.com/extjs/4.2.3/#!/api/Ext.form.action.Action-cfg-timeout